EnglishFilter/sections/specific.txt

```diff
@@ -161,6 +161,7 @@ allcelebs.club##table[align="center"][style="width: 700px;"]
 allcelebs.club##a[href^="http://www.searchcelebrityhd.com/"]
 ettv.to##.vp_ab
 ettv.to##.imdb > div[class^="download_liinks"]
+||cryptobrowser.site^$domain=subdl.com
```

Comment:
I can't reproduce this ad today, but as I recall it was something like `get.cryptobrowser.site/promo/ad/` which is used on other websites too - https://publicwww.com/websites/%22get.cryptobrowser.site%2Fpromo%2Fad%2F%22/
So, I think that it will be better to add something like:
`||get.cryptobrowser.site/promo/ad/`

Reply:
agree

Reply:
changed rule to `||get.cryptobrowser.site/promo/ad/$domain=subdl.com`

Reply:
Please remove `$domain=subdl.com` and just add `||get.cryptobrowser.site/promo/ad/`, because similar ads are also on other websites, for example here - http://kumpulfilemgratis.blogspot.com/ (at the bottom) and here - https://imsiservice.com/

Reply:
@AdamWr moved rule to cryptominers.txt

Reply:
I think that it's "normal" ad and it's not mining, so maybe it would be better to leave it in `specific.txt`?